crates/uv/src/commands/tool/run.rs

```diff
@@ -183,12 +185,17 @@ pub(crate) async fn run(
 
     // We check if the provided command is not part of the executables for the `from` package.
     // If the command is found in other packages, we warn the user about the correct package to use.
-    warn_executable_not_provided_by_package(
-        executable,
-        &from.name,
-        &site_packages,
-        invocation_source,
-    );
+    match &from {
```

Comment:
Do we need to do anything to ensure that `python` resolves to the "correct" Python? Is there any risk that the user does, like, `uvx --python 3.7 python` and we end up picking a Python other than the discovered interpreter?

Reply:
I did not consider that; I will add a guard.

Reply:
Opted not to do this as it complicates the implementation and I'm pretty sure there can't be a problem in-practice. We do the same thing for tools.